client/post-editor/editor-notice/index.jsx

```diff
@@ -89,6 +89,31 @@ export class EditorNotice extends Component {
 					comment: 'Editor: Message displayed when a post is published, with a link to the site it was published on.'
 				} );
 
+			case 'scheduled':
```

Comment:
Probably not going to handle it in this PR, but perhaps we need to add logic in the notices to support non page/post post types.  Though translations could be a challenge.